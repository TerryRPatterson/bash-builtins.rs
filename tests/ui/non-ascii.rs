use bash_builtins::BuiltinOptions;

#[derive(BuiltinOptions)]
enum Opt {
    #[opt = 'Ω']
    Ω
}

fn main() {}
