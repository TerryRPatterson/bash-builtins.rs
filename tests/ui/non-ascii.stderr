error: #[opt] requires an ASCII alphanumeric character
 --> $DIR/non-ascii.rs:5:5
  |
5 |     #[opt = 'Î©']
  |     ^
